src/App.tsx

```diff
@@ -1,39 +1,29 @@
-import React from 'react';
-import './App.scss';
+import { FC, useState } from 'react';
+import './styles/App.scss';
 import Carousel from './components/Carousel';
 
-interface State {
-  images: string[];
-}
+const App: FC<{}> = () => {
+  const [images] = useState([
+    './img/1.png',
+    './img/2.png',
+    './img/3.png',
+    './img/4.png',
+    './img/5.png',
+    './img/6.png',
+    './img/7.png',
+    './img/8.png',
+    './img/9.png',
+    './img/10.png',
+  ]);
 
-class App extends React.Component<{}, State> {
-  state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
-  };
+  return (
+    <div className="App">
+      {/* eslint-disable-next-line */}
+      <h1 className="App__title" data-cy="title">Carousel with {images.length} images</h1>
```

Comment:
not sure what linter complained about, but i'd multi-line it
```suggestion
      <h1 className="App__title" data-cy="title">
        Carousel with {images.length} images
      </h1>
```